channel-transport/fluid-openfoam/run.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+set -e -u
+
+blockMesh
+touch fluid-openfoam-pimplefoam.foam
```

Comment:
```suggestion
touch fluid-openfoam.foam
```

(probably copy-paste from the partitioned pipe tutorial, where we have more than one OpenFOAM cases)